### PR TITLE
LPS-151369, LPS-149366, LPS-152997 | master

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-151369
https://issues.liferay.com/browse/LPS-149366
https://issues.liferay.com/browse/LPS-152997

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro [hunk 1]
```diff
@@ -58,6 +58,64 @@ definition {
...
+		if (!(isSet(skipNavigation))) {
+			DMNavigator.gotoAddDocumentTypeCP(dmDocumentTypeName = "${dmDocumentTypeName}");
+		}
+
+		Type(
+			key_text = "${key_text}",
+			locator1 = "TextInput#ANY",
+			value1 = "${videoURL}");
+
+		Pause(locator1 = "5000");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro:26]: @liuxingye @WYZ-Issac  is this necessary? Is there a way we can use something other than a pause?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/video/ExternalVideo.testcase [hunk 2]
```diff
@@ -35,6 +35,30 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-151368. It checks that the video URL is saved when adding external video shortcut in non-default language."
+	@priority = "4"
+	test CanExternalVideoURLSavedAfterLoginNonDefaultLanguage {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/video/ExternalVideo.testcase:40]: Change test name to ExternalVideo#VideoURLIsSavedWhenUsingNonDefaultLanguage

File: portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro (hunk 1 above)
Comment (liuxingye) [reply portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro:26]: @austinchiang I have removed the new macro

Comment (austinchiang) [reply portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro:26]: woops missed it. Thanks!

Comment (liuxingye) [reply portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro:26]: it is ok

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

**Comment by austinchiang** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/3019

**Pull request closed without merging**